addons/library.xbmc.addon/libXBMC_addon.h

```diff
@@ -47,6 +47,8 @@ typedef intptr_t      ssize_t;
 #define ADDON_HELPER_ARCH       "powerpc-osx"
 #elif defined(__arm__)
 #define ADDON_HELPER_ARCH       "arm-osx"
+#elif defined(__aarch64__)
+#define ADDON_HELPER_ARCH       "arm-osx"
```

Comment:
Well that's what I'm trying to say basically , if you think this code is okay like this.